RecoTracker/LSTCore/src/alpaka/NeuralNetwork.h

```diff
@@ -39,102 +39,88 @@ namespace lst::t5dnn {
 
   template <typename TAcc>
   ALPAKA_FN_ACC ALPAKA_FN_INLINE float runInference(TAcc const& acc,
```

Comment:
```suggestion
  ALPAKA_FN_ACC ALPAKA_FN_INLINE bool runInference(TAcc const& acc,
```
to match what's actually returned